plastic_tui/src/ui.rs

```diff
@@ -537,6 +534,21 @@ impl Ui {
             }
         }
 
+        disable_raw_mode().unwrap();
+        execute!(
+            io::stdout(),
+            Show,
+            LeaveAlternateScreen,
+                       if let Some(ref mut player) = self.audio_player {
+                let audio_buffer = process_audio(&audio_buffer, 1.0);
+                player.queue(&audio_buffer);
+            }
+
+            if let Some(remaining) = fps.remaining() {
+                thread::sleep(remaining);
+            }
+        }
+
```

Comment:
This got duplicated and is causing build errors

Reply:
Bruh how did that even happen. Lol